### ISSUE TITLE
Make kaobook more configureable

### ISSUE THREAD

**Issue by AlexanderZeilmann** [fmarotta/kaobook#209]
## Summary

Allow more configurations using package options.

## Details
Currently, when I want to make changes to kaobook (kaobiblio, kaorefs, kaotheorems, ...) I have to copy the corresponding file and make the changes. This, however, brings the file out of sync with any new features that are developed in kaobook.
Therefore I would like to have kaobook more configureable, e.g., if I don't want to have the backrefs in the bibliography it should be possible to just write:
```latex
\usepackage[backref=false]{kaobiblio}
```

It seems that such a feature existed at some point as the documentation says:

> For instance, if you want to use the ‘philosophy-modern’ style for your bibliography, you might have something like this in the preamble:
> ```latex
> \usepackage[style=philosophy-modern]{styles/kaobiblio}
> ```

Note that the documentation still contains a reference to the old `styles` folder.

**Comment by fmarotta** [comment]
I agree, copying the file and making changes is not something that I'd like users to do on a regular basis. Admittedly, I used to suggest this practice in the beginning, and often it is the only way to do something because of the lack of configurability in kaobook, but this cannot be an option if we want to submit kaobook to CTAN. I'll definitely try to add more options...

However, `backref` in kaobiblio works for me. If I use `\usepackage[backref=false]{kaobiblio}`, I don't get the back references. Not sure what's going wrong for you: if you still have this problem, could you provide an MWE? Also, suggestions about which options to implement are most welcome.

**Comment by AlexanderZeilmann** [comment]
You are absolutely right, `\usepackage[backref=false]{kaobiblio}` is working, I must have made some changes to my local kaobook that destroyed the functionality 😕 

I will also think about settings that should be configureable.

**Comment by pi8027** [comment]
FYI, I had to comment out the following line. I hope it will become configurable. https://github.com/fmarotta/kaobook/blob/2a82b3a8a3556d646d47bd719829d6d65e34895b/kaobiblio.sty#L68
